src/Tools/AnalyzerRunner/AnalyzerRunner.csproj

```diff
@@ -11,6 +11,17 @@
     <!-- Automatically generate the necessary assembly binding redirects -->
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
❗️ Remove the `Condition` here so it applies to all builds

Reply:
sure thing